tests/integration/test_dataapi.py

```diff
@@ -59,30 +52,100 @@ def _build_base_url(self):
 
         return base_url
 
-
     def test_real_request(self):
         """Make a request against Data Api to ensure the SDK works"""
         client = DataApi()
-        res = client.request(self._build_base_url() + endpoint, security, consumer_secret, request,
+
+        test_url = self._build_base_url() + items_endpoint
+
+        print("test_url is", test_url)
```

Comment:
Best to cleanup this print statement and the extra blank lines around it.

All the extra whitespace added in this function can make for a bit of a messy diff when there weren't actually any logical changes in it. Plus all the extra spaces make it a little more difficult to differentiate between the ends of functions. We probably don't want a new line after every single line of code through here. 😅

Reply:
Yeah I think I went a bit newline happy... I should have reverted those local changes before committing my actual changes.